sympy/stats/drv.py

```diff
@@ -45,8 +46,10 @@ def __new__(cls, *args):
     def check(*args):
         pass
 
-    def sample(self):
+    def sample(self,size=()):
```

Comment:
Please provide some examples of using this in your PR in a reply to this comment.